client/network/test/src/lib.rs

```diff
@@ -662,7 +672,10 @@ pub struct FullPeerConfig {
 	pub sync_mode: SyncMode,
 }
 
-pub trait TestNetFactory: Sized where <Self::BlockImport as BlockImport<Block>>::Transaction: Send {
+pub trait TestNetFactory: Sized
```

Comment:
I think I'd drop this and the formatting changes below.

Reply:
undone.